To Do Bot/README.md

```diff
@@ -0,0 +1,2 @@
+# ToDoBot
```

Comment:
Please add `Usage` , `requirements`.
See other script's for reference.

Reply:
Added as per your suggestion.